src/scripts/installer-queries/data-views.sql.fmt

```diff
@@ -127,3 +127,4 @@ SELECT V:RUN_ID::VARCHAR AS run_id
 FROM results.query_metadata
 WHERE V:QUERY_NAME ILIKE '%_VIOLATION_SUPPRESSION'
 ;
+
```

Comment:
not sure what happened here, but I'm almost certain we want a single newline at end of file as this might break things in the runner

Reply:
fixed